inc/settings.php

```diff
@@ -278,7 +280,7 @@ public function parse_settings( $new_settings ) {
 					$sanitized_value = $this->to_bound_integer( $value, 100, 5000 );
 					break;
 				case 'quality':
-					$sanitized_value = $this->to_bound_integer( $value, 1, 100 );
+					$sanitized_value = $this->to_bound_integer( $value, 50, 100 );
```

Comment:
I think here is the most important thing, the initial issue with value being saved as one is because `quality` can be a string, `auto`, or number, to which the sanization does not consider the string case, thus the value will be saved as `1`, it is left untouched.

We should handle this case.